pandas/core/generic.py

```diff
@@ -10460,7 +10460,7 @@ def _add_series_or_dataframe_operations(cls):
         Add the series or dataframe only operations to the cls; evaluate
         the doc strings again.
         """
-        from pandas.core.window import EWM, Expanding, Rolling, Window
+        from pandas.core.window import EWF, Expanding, Rolling, Window
```

Comment:
i would be ok with actually writing this classname out if we are going to change it, e.g. ExponentialMoving or ExponentialMovingWindow

Reply:
sounds good might as well

Reply:
quick one could you clarify what you mean by writing it out?

Reply:
I _think_ @jreback meant you could rename `EWM` to `ExponentialMoving` in `core.window.ewm.py` where the class is created

Reply:
k

Reply:
changes resolved